README.md

```diff
@@ -1,11 +1,55 @@
-# Your GitHub Learning Lab Repository for Community Starter Kit
+# welcome bot: A Probot App
 
-Welcome to **your** repository for your GitHub Learning Lab course. This repository will be used during the different activities that I will be guiding you through to learn about making your project friendlier for collaboration.
+DESCRIPTION PLACEHOLDER
```

Comment:
+ A handy app that gives new users warm and welcoming messages
+ Customize-able messages 
+ Open sourced like the millions of wonderful programs here on Github